src/predict/api/tests.py

```diff
@@ -5,6 +5,43 @@
 from mongomock import MongoClient
 
 from app import create_app
+from prediction import validate_param_values
+
+valid_params = [
+    {"correlation_fault": "0", "missing_data_fault": "0"},
+    {"correlation_fault": "1", "missing_data_fault": "1"},
+    {"correlation_fault": "0", "missing_data_fault": "1"},
+    {"correlation_fault": "1", "missing_data_fault": "0"},
+]
+
+invalid_params = [
+    {"correlation_fault": "2", "missing_data_fault": "0"},
+    {"correlation_fault": "0", "missing_data_fault": "-1"},
+    {"correlation_fault": "a", "missing_data_fault": "b"},
+    {"correlation_fault": "", "missing_data_fault": ""},
+]
+
+
+@pytest.fixture(params=valid_params)
+def valid_param(request):
+    return request.param
+
```

Comment:
thanks @Mnoble-19 , there is an opportunity to add more test cases to this. 

- In each of our future PRs, we need to incrementally add more test cases to increase our test coverage.
- There is also an opportunity to review the file organisation of our test cases